lua-functions.cc

```diff
@@ -20,12 +20,12 @@ int listDirectoryFull(lua_State *L, int mode) {
 	WIN32_FIND_DATAA fdFile;
 	HANDLE handle = NULL;
 
-	std::string path = format("%s\\*.*", dirname);
+	std::string path = format(".\\%s\\*.*", dirname);
 	if((handle = FindFirstFileA(path.c_str(), &fdFile)) == INVALID_HANDLE_VALUE) {
 		return 1;
 	}
 
-	int index = 0;
+	int index = 1;
```

Comment:
I assume this fixes the results of `os.listdirs`/`files` being 0-indexed?

Reply:
Yes, that is correct.